nalaf/download_data.py

```diff
@@ -13,6 +14,14 @@
 if __name__ == '__main__':
 
     # Download NLTK models
+    try:
+        _create_unverified_https_context = ssl._create_unverified_context
+    except AttributeError:
+        pass
+    else:
+        ssl._create_default_https_context = _create_unverified_https_context
```

Comment:
What is this? Do we really need it?

Do we need it for the downloading of the NLTK models?

Reply:
yes, it is required. without this, we have to manually download it and put it to the corresponding directory.